config/i18n.json

```
@@ -583,6 +583,25 @@
     "MissingClassDescription": "The build you're trying to view is for a character class you don't have.",
     "NewEmptyLoadout": "Create Empty Loadout",
     "NoBuildsFound": "No builds found. Try expanding your filtering criteria.",
+    "NoBuildsFoundExplainer": {
+      "Header": "No builds found. Here are possible reasons DIM couldn't find any sets:",
+      "AlwaysInvalidMods": "These mods don't fit into any of your items:",
+      "RemoveMods": "Remove these mods",
+      "MaybeRemoveMods": "Consider removing mods",
+      "AssumptionsRestricted": "DIM is restricted in its allowed assumptions about armor energy",
+      "AssumeElementChange": "Assume armor will change energy type",
+      "AssumeMasterworked": "Assume armor is masterworked",
```

```suggestion
      "AssumeMasterworked": "Allow DIM to recommend upgrading armor to Masterwork",
```
this or maybe 
"Allow DIM to recommend masterworking armor",